decko/spec/cypress/integration/edit/nest_editor.spec.coffee

```diff
@@ -0,0 +1,120 @@
+open_nest_editor = () ->
+  cy.get("[title='Insert/edit nest']").click()
+
+nest_option = (section, row, name, value) ->
+  cy.get("._options-select").eq(section)
+    .find("select._nest-option-name").eq(row).select2(name)
+  cy.get("._options-select").eq(section)
+    .find("input._nest-option-value").eq(row)
+    .type value
+
+type_nest = (text) ->
+  cy.tinymce (ed, win) ->
+    ed.focus()
+    t = text.replace("{cursor}", "<span id='mymarker'>\u200b</span>")
+    ed.insertContent(t)
+    marker = win.jQuery(ed.getBody()).find('#mymarker')
+    ed.selection.select(marker.get(0))
+    marker.remove()
+    ed.insertContent(" ")
+
+describe 'nest editor', () ->
+  before ->
+    cy.login()
+    #cy.clear_script_cache()
+
+  specify "nest editor", () ->
+    cy.ensure "nests", ""
+    cy.visit_bridge "nests"
+    open_nest_editor()
+
+    cy.get "#nest_name"
+      .type "NaNa"
+    cy.contains "button", "Configure items"
+      .click()
+
+    cy.contains "h6", "items"
+    cy.contains "button", "Configure subitems"
+
+    cy.get("._options-select").eq(0).as("options")
+    cy.get("._options-select").eq(1).as("itemoptions")
+
+    nest_option(0, 1, "title", "T")
+    nest_option(1, 1, "show", "IS")
```

Comment:
"IS" doesn't mean anything here, right?  just a fake view name?

Reply:
yes, it means "[I]tem [S]how"